### PR TITLE
Rewording 'coming soon' statement

### PR DESCRIPTION
**Pull request by bgribaudo**
It's been well more than the several weeks that the existing wording implies. :-) Rewording to not promise a specific timeframe.

### CODE REVIEW

**Comment by Court72** [comment]
@DougKlopfenstein, @bezhan-msft 

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Review by ehrenMSFT — APPROVED** [review]
I guess it depends on how many weeks. ;)

**Comment by ehrenMSFT** [comment]
#sign-off

**Comment by Jak-MS** [comment]
@bezhan-msft @DougKlopfenstein - please sign off. thanks.

File: powerquery-docs/data-privacy-firewall.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ ms.author: ehvonleh
 # Behind the scenes of the Data Privacy Firewall
 
 >[!NOTE]
->Privacy levels are currently unavailable in Power Platform dataflows. The product team is working towards re-enabling this functionality in the coming weeks.
+>Privacy levels are currently unavailable in Power Platform dataflows. The product team is working towards enabling this functionality in the future.
```
Comment (Jak-MS) [inline powerquery-docs/data-privacy-firewall.md:13]:
```suggestion
>Privacy levels are currently unavailable in Power Platform dataflows, but the product team is working towards enabling this functionality.
```
upon re-evaluation, the specific future promise is not permitted in any case. proceeding with merging this (after a slight edit).

**Comment by bgribaudo** [comment]
Thanks, @Jak-MS!

**Pull request merged**